3D Chess/Assets/Scripts/PieceMovement.cs

```diff
@@ -0,0 +1,39 @@
+using UnityEngine;
+
+public class PieceMovement : MonoBehaviour
```

Comment:
Funnily enough it was called `MovementManager` before. I don't like calling everything a manager (`GameState` was `GameManager`, there is an `EventManager`, there was a `SpawnManager`). It's kind of redundant because every class manages something.

But then again, naming things is one of the hardest problems in software engineering so I'm open for suggestions. ;)

Rethinking the names, how about:
- `Events` (instead of `EventManager`) - since it's actually just a list of events anyway
- `GameState` (as it is right now already)
- `PieceSpawner` (seems fine too I assume)
- `InputListener` (instead of `PieceMovement`)

--> I have restructured some things. I think the architecture overall is better now too but it also avoid the name manager all together. See 9796f61.
Also, `Events` is now no longer a prefab but instead we do a look-up in `Square` and `Piece` to achieve what was a bit less readable before:
```
private void Awake()
    {
        events = FindObjectOfType<Events>();
    }
```

What do you think of this commit?